server/gokbg3/grails-app/controllers/org/gokb/rest/ReviewsController.groovy

```diff
@@ -408,7 +497,100 @@ class ReviewsController {
       linksObj.update = [href:base]
       linksObj.delete = [href:base]
     }
-
     return linksObj
   }
+
+
+  private def getEscalationTargetGroupId(def rrId, def activeGroupId, def params){
+    def result = ['result':'ERROR', 'isEscalatable':false, 'params': params]
+
+    def rr = ReviewRequest.get(genericOIDService.oidToId(rrId))
+    if (!rr){
+      response.setStatus(404)
+      result.message = "ReviewRequest not found for id ${rrId}."
+      return result
+    }
+    if (!rr.isEditable()){
+      response.setStatus(403)
+      result.message = "ReviewRequest for id ${rrId} may not be edited."
+      return result
+    }
```

Comment:
This is a private method. Permissions are checked in the two calling methods, `escalate()` and `isEscalatable()`. Should be good like this. Otherwise, please specify and reopen conversation, @hornmo .